section2/exercises/if-statements.js

```diff
@@ -87,10 +104,23 @@ var hasSauce = true;
 
 // Write your conditional here
 
+if ((cupsOfFlour === 1) && (hasSauce === true)) {
```

Comment:
1. Is there a way you could rework this so that you don't have any redundant lines of code?
2. If the `hasSauce` variable is storing the Boolean of `true`, then `hasSauce === true` and `hasSauce` will both evaluate to `true`. SO on lines 111 and 113, you could get rid of the `=== true` to shorten a bit. Not necessary, but just thought I'd share since you are in a very solid place. It's pretty common to see that in others' code, so always good to know when you can do something more than one way.